### ISSUE TITLE
Winlink hybrid: Update MPS list

### ISSUE THREAD

**Issue by martinhpedersen** [la5nta/pat#51]
For users to fully take advantage of the Winlink Hybrid Network, they would need a way to update their MPS (Message Pickup Station) lists.

Winlink Express updates the MPS list via the Winlink Web API directly over HTTP, or by posting a special INQUIRY message containing a sequence of these API requests.

These updates (requests) are guarded by a ```WebServiceAccessCode``` which we need to obtain/generate in order for users to update their MPS list via Pat.

Example update message:
>```
>Message ID: U2YDOUIYDUA4
>Date: 2016/03/27 14:52
>From: LA4TTA
>To: INQUIRY 
>Source: LA4TTA
>Subject: CMSWEBSVC
>
>/mps/delete?requester=LA4TTA&callsign=LA4TTA&WebServiceAccessCode=F630KKDH9L
>/mps/add?requester=LA4TTA&callsign=LA4TTA&mpsCallsign=LA3F&WebServiceAccessCode=F630KKDH9L
>/mps/add?requester=LA4TTA&callsign=LA4TTA&mpsCallsign=SM5RVH&WebServiceAccessCode=F630KKDH9L
>/mps/add?requester=LA4TTA&callsign=LA4TTA&mpsCallsign=HB9AW&WebServiceAccessCode=F630KKDH9L
>/mps/delete?requester=LE1OG&callsign=LE1OG&WebServiceAccessCode=F630KKDH9L
>/mps/add?requester=LE1OG&callsign=LE1OG&mpsCallsign=LA3F&WebServiceAccessCode=F630KKDH9L
>/mps/add?requester=LE1OG&callsign=LE1OG&mpsCallsign=SM5RVH&WebServiceAccessCode=F630KKDH9L
>/mps/add?requester=LE1OG&callsign=LE1OG&mpsCallsign=HB9AW&WebServiceAccessCode=F630KKDH9L
>```
>

*The original WebServiceAccessCode have been changed, in case it contains sensitive information. Thanks to LA4TTA for providing this example.*

Until this is implemented users are advised to update their MPS list using Winlink Express.

**Comment by martinhpedersen** [comment]
Our new API key is authorized for the following APIs:
* MessagePickupStationAdd
* MessagePickupStationDelete
* MessagePickupStationGet
* MessagePickupStationList

Ref issue #110

**Comment by AquaeAtrae** [comment]
@martinhpedersen Is it still the case that that PAT has Limited support for "Radio-only forwarding network support" because it "can't update MPS"? 
https://winlink.org/ClientSoftware

**Comment by martinhpedersen** [comment]
Yes, unfortunately. It would probably be a relatively small task to implement a CLI command to do this. Any help would be appreciated 😊

**Comment by jboverfelt** [comment]
From the PR:

- I think I like `--via-radio` better as the flag. The user probably cares more about how it's being sent rather than the implementation detail of it being an INQUIRY message
- I think enriching rmslist is a great idea and probably better than adding flags to mps list
- We probably don't need to go so far as to parse the email result for the INQUIRY messages at least at first. While I want to fully support offline updates for this, I do still think most people will set it up once and not touch it much.

I updated the checklist to reflect the PR too!

**Comment by martinhpedersen** [comment]
This all sounds good to me @jboverfelt 😊

**Comment by jboverfelt** [comment]
So glad to see the initial support in the latest release! I'm using it now. I haven't had time to start on any of the todos from this issue. I'll let you know if I'm able to get started. If you get some time before then feel free to give it a go!
